index.html

```diff
@@ -44,4 +44,171 @@ <h1 class="headline-heading heading-large">I'm Mariane <br> Glad to see you!</h1
       </ul>
     </div>
   </section>
-  <!-- Headline Section Ends Here -->
\ No newline at end of file
+  <!-- Headline Section Ends Here -->
+
+  <!-- Works Section Starts Here -->
+  
+  <section id="portfolio" class="projects">
+    <!-- Card 1 starts -->
+    <ul class="works-cards">
+      <li class="works-card">
+        <img src="./Img/snapshoot-portfolio.png" alt="Recent Work">
+        <div class="works-card-main">
+          <h3 class="works-card-heading-large">Tonic</h3>
+          <div class="works-card-client">
+            <p class="paragraph">CANOPY</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-role paragraph">Back End Dev</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-year paragraph">2015</p>
```

Comment:
- [ ] You have used fontawsome `icon` with `<i>` tag but haven't linked the `fontawsome` in `<head>` tag of the HTML file. Thats why `<i>` tag is not working properly. Kindly fix that issue.
- You can consider follow this [article](https://www.w3schools.com/icons/fontawesome_icons_intro.asp).
- Or you can copy the images given in the template and use it using `img` tag, you don't need fontawesome then.
- ![image](https://user-images.githubusercontent.com/60083437/235313251-5206e297-f22c-4fd4-aa7f-07edc3c7c82f.png)